Improve description of the fair process directive (#248)
nextflow-io/training: docs/basic_training/cache_and_resume.md

    | PROCESS_C
```

> If meta maps are not possible, an alternative is to use the [`fair`](https://nextflow.io/docs/edge/process.html#fair) process directive. When this directive is specified, Nextflow will guarantee that the order of outputs will match the order of inputs. It's important to mention that the order in which the tasks will be finished will not necessarily follow the order in the input channel, but Nextflow guarantees that, by the end of it, the output channel will contain the elements in the respective order.

Reviewer: ```suggestion
If meta maps are not possible, an alternative is to use the [`fair`](https://nextflow.io/docs/edge/process.html#fair) process directive. When this directive is specified, Nextflow will guarantee that the order of outputs will match the order of inputs (not the order in which the tasks run, only the order of the output channel).
```
